src/QsCompiler/LanguageServer/ProjectLoader.cs

```diff
@@ -54,7 +54,7 @@ internal class ProjectLoader
             new(@"(netstandard[1-9]\.[0-9])|(netcoreapp[1-9]\.[0-9])|(net[1-9][0-9][0-9]?)|(net[1-9]\.[0-9])");
 
         private readonly ImmutableArray<string> supportedQsFrameworks =
-            ImmutableArray.Create("netstandard2.", "netcoreapp2.", "netcoreapp3.", "net6.");
+            ImmutableArray.Create("netstandard2.", "netcoreapp3.1", "net6.");
```

Comment:
Could this potentially impact any existing code using 3.0?

Reply:
I don't know... this change is already in main and is just being brought over to the feature branch now. It was made as part of https://github.com/microsoft/qsharp-compiler/pull/1455.